grp/simple.gi

```diff
@@ -233,9 +233,46 @@ local p,f,z,G,o;
       [0,0,0,0,1,0,0]]);
   fi;
 
+  SetSize(G,q^6*(q^6-1)*(q^2-1));
```

Comment:
Just curious: Is this a 'well known' thing?

Reply:
@wilfwilson Yes. (Though I guess the "well" is relative, given that you ask). The order of every Chevalley group is a polynomial in the field order. This follows from the general theory and is proven e.g. in Carter's (simple groups) book. I copied the formula from a table in the ATLAS preface.

Reply:
Thanks for the answer, that’s good to know. Despite my interest in finite simple groups, I’m not at all well-versed in their details, so the fact that I didn’t know shouldn’t weigh in on the “well-known”-ness of this fact. My algebra education focused more on semigroups, rings, fields, and infinite groups, and not on the finite ones. 🙂 I still lack quite a few of the basics.